proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala

```diff
@@ -52,28 +52,35 @@ object WKT {
     */
   def getEpsgCode(wktString: String): Option[Int] = {
     val wktParsed = WKTParser(wktString)
-    parsed.find{
-      case (_, wkt) => wkt == wktParsed
-    }.map(_._1)
+    val db = parsed.find { case (_, wkt) => wkt == wktParsed }.map(_._1)
+    if(db.nonEmpty) db
+    else
+      wktParsed match {
+        case wkt: ProjCS =>
+          wkt.extension.flatMap {
+            case ExtensionProj4(proj4String) =>
```

Comment:
That is why it is convinient to have `ExtensionProj4`

Reply:
You're just saying its convenient to be able to `match` on the ExtensionProj4 case class right?

Reply:
Yep!